.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node CI
```

Comment:
This is from master (https://github.com/prezly/javascript-sdk/commit/1182f85307a1b2863b0948b69c8bdfb5b1ecf474), but missing in the target branch.

It's automatically generated by the Github Actions interface. Runs `npm run build` and `npm test` in node@8, node@10, and node@12 (current LTS vesions).